packages/spindle-hooks/src/useTimeDistance/formatDistance.ts

```diff
@@ -0,0 +1,114 @@
+const getFullYear = (d: Date) => d.getFullYear();
+const getMonth = (d: Date) => d.getMonth() + 1;
+const getDate = (d: Date) => d.getDate();
+const getHours = (d: Date) => d.getHours();
+const getMinutes = (d: Date) => d.getMinutes();
+const getStringDay = (d: Date) => {
+  const idx = d.getDay();
+  switch (idx) {
```

Comment:
[nits] ↓で良さそうかなと思いました（些細なのでどちらでもいいんですが）。
```suggestion
  switch (d.getDay()) {
```